3ds_pdn.rsf

```diff
@@ -0,0 +1,64 @@
+BasicInfo:
+  Title                   : pdn
```

Comment:
Please change this to 3ds_pdn or something, this is really helpful to know if the module loaded or not due to user error

Reply:
Done